combiner/combiner_test.go

```diff
@@ -0,0 +1,33 @@
+package combiner_test
```

Comment:
I think there should be some tests in addition to the example :)
At least before merging into `master`. But I can implement them later in case you're busy.